ProcessLib/LIE/Common/LevelSetFunction.cpp

```diff
@@ -10,14 +10,22 @@
 
 #include <boost/math/special_functions/sign.hpp>
 
+#include "BranchProperty.h"
 #include "FractureProperty.h"
+#include "JunctionProperty.h"
 
 namespace
 {
 // Heaviside step function
 inline double Heaviside(double v)
 {
-    return (v < 0.0) ? 0.0 : 1.0;
+	return (v < 0.0) ? 0.0 : 1.0;
```

Comment:
It seems, that the clang-format is not using the `.clang-format` from the source root...